examples/handle_quit.rs

```diff
@@ -0,0 +1,18 @@
+use macroquad::prelude::*;
+
+fn window_conf() -> Conf {
+    Conf {
+        window_title: "Handle quit event".to_owned(),
+        handle_quit_event: true,
+        ..Default::default()
+    }
+}
+
+#[macroquad::main(window_conf)]
+async fn main() {
+    loop {
+        clear_background(WHITE);
+        if is_quit_requested() { break; }
```

Comment:
Maybe do a `cancel_quit`(not sure about naming) API call available? To be able to do a nice  "Are you sure?" dialogue